Assignment3.txt

```diff
@@ -0,0 +1,15 @@
+Ashank Kumar
+
+1a. O(n)
+
+1b. O(n^2)
+
+1c. O(n/2)(log2(n))
```

Comment:
almost! it's just nlog(n) since it's big O notation. you should give explanations for your answers!